frontend/device/kobo/device.lua

```diff
@@ -534,6 +536,83 @@ local KoboCondor = Kobo:extend{
     isSMP = yes,
 }
 
+-- Kobo Clara B/W:
+local KoboSpaBW = Kobo:extend{
+    model = "Kobo_spaBW",
+    isMk7 = yes,
+    hasEclipseWfm = yes,
+    canToggleChargingLED = yes,
+    led_uses_channel_3 = true,
```

Comment:
This is *probably* unnecessary (if I trust the Elipsa 2E's current devcaps).

Reply:
This is supposed to do something with the charging LED, right? If yes, then nothing changes with or without, I have no light in KOReader, while I do with the Kobo default software.

Reply:
Right, we're probably also missing discovery of the right led (i.e., probably `/sys/class/leds/LED` as `self.charging_led_sysfs_knob`).